cmd/egnaddrs/README.md

```diff
@@ -0,0 +1,41 @@
+## egnaddrs
+This tool is used to help debug and test eigenlayer/avs deployments and contract setups.
+
+### How to install
+#### Install from repository
+```bash
+   go install github.com/Layr-Labs/eigensdk-go/cmd/egnaddrs@latest
+```
+
+#### Install from local source
+If you have git cloned eigensdk-go to your machine, navigate to [cmd/egnaddrs](.) directory and run
+```bash
+   go install
+```
+
+### Usage
+
+Currently egnaddrs only supports deriving contract addresses starting from the registry-coordinator address. It then prints the following datastructure:
+
+```bash
+$$$ egnaddrs --registry-coordinator 0x9E545E3C0baAB3E08CdfD552C960A1050f373042
```

Comment:
can we also add one where we can input AVS service manager contracts and get the rest of them.

Reply:
Done [7a60f64](https://github.com/Layr-Labs/eigensdk-go/pull/50/commits/7a60f64b9ecda1723dba68c663c2e5e144bacc52)

Also added tests in [6b29849](https://github.com/Layr-Labs/eigensdk-go/pull/50/commits/6b29849684b84bb421658429f69d7afec2033bed), please have a look